cryptol/REPL/Haskeline.hs

```diff
@@ -67,12 +75,18 @@ crySession mbBatch stopOnError =
            | all (all isSpace) ls -> loop (lineNum + length ls)
            | otherwise            -> doCommand lineNum ls
 
+  run lineNum cmd =
+    case replMode of
+      InteractiveRepl    -> runCommand lineNum Nothing cmd
+      Batch path         -> runCommand lineNum (Just path) cmd
+      InteractiveBatch _ -> runCommand 1 Nothing cmd
```

Comment:
Do we really want every line to have line number 1 when running with `InteractiveBatch`?  It would be easy to count up inside a single interactive batch file, which is how the system would actually behave if you typed in the lines sequentially at the prompt...

Reply:
When I type in sequentially at the prompt, I get the line number as `1` every time.
```
Cryptol> 1
Showing a specific instance of polymorphic result:
  * Using 'Integer' for the type of '(expression)'
1
Cryptol> a

[error] at <interactive>:1:1--1:2 Value not in scope: a
```

This is the behavior I'm trying to mimic. Not sure if it's going to be useful for anything other than the exercise checker, but that's the functionality I'd need for it to be used for that. 

If this change doesn't make sense for cryptol itself, I could do something else -- namely, have the checker itself ignore those line numbers. But this feature of cryptol would make it a lot easier.

Reply:
This is what you should see in current master:
```
Loading module Cryptol
Cryptol> 1
Showing a specific instance of polymorphic result:
  * Using 'Integer' for the type of '(expression)'
1
Cryptol> a

[error] at <interactive>:2:1--2:2 Value not in scope: a
Cryptol> a

[error] at <interactive>:3:1--3:2 Value not in scope: a
```

Reply:
Nevermind -- that was my mistake. I was running an old version of the executable. Let's keep the line numbers incrementing.